Jenkinsfile

```diff
@@ -16,20 +16,23 @@ def baseConfig = {
 
 def defaultParams = [
     booleanParam(name: 'RELEASE_BUILD',
-        description: 'Is this a release build.',
+        description: 'Is this a release build. If so, GIT_REVISION must be specified, and only on-prem images will be built.',
         defaultValue: false),
     string(name: 'GIT_REVISION',
```

Comment:
It doesn't have to be a SHA, it can also be a branch name or Git tag. However, directly using the tag from a previous build doesn't work since the tags have CP dependencies updated to be beta packaging builds, which means the job fails to process the pom since the job isn't configured with the packaging beta maven repo.

(Tried just now to confirm and the error was
```
20:30:04  [FATAL] Non-resolvable parent POM for io.confluent.ksql:ksqldb-parent:0.8.0: Could not find artifact io.confluent:rest-utils-parent:pom:6.0.0-beta200310175819 in confluent (https://****-confluent-packages-beta-maven.s3-us-west-2.amazonaws.com/6.0.0-beta200320180305/1/maven) and 'parent.relativePath' points at wrong local POM @ line 22, column 13
```
)

Reply:
The way we'd reproduce an earlier build is to go to the Git tag, find the parent SHA, and specify that SHA in the build.